docs/.vitepress/config.ts

```diff
@@ -15,8 +15,8 @@ export default defineConfig({
     },
     
     nav: [
-      { text: 'Guide', link: '/core-concepts' },
-      { text: 'API', link: '/api' }
+      { text: 'Guide', link: '/core-concepts/defining-routes' },
```

Comment:
I feel like the intro or getting started pages are better entry points for the guide